{"uid":"d141263e4b68c05e","name":"Create a new Order for new Customer","fullName":"TestOrder.TestOrder#test_create_new_user_order_1___frugal_latido_com_np____Test_123____9881012071____abced_Test_1____testuser171_mail_com____2023_05_28____Siliguri____male____Ek__male_____Black_Sheep____38____Silver____Black____No____40___5_6__4_4__5__46__8__12__11__7__1__9___Fit____Test_001____2023_05_29___100900_","historyId":"9d63841389ef80899607cf2352ff8924","time":{"start":1685022470210,"stop":1685022773172,"duration":302962},"status":"failed","statusMessage":"AssertionError: None != 'Order Succesfully Created' : Verification Failed to Create a new Order.","statusTrace":"self = <TestsCases.TestOrder.TestOrder testMethod=test_create_new_user_order_1___frugal_latido_com_np____Test_123____98810120...p____38____Silver____Black____No____40___5_6__4_4__5__46__8__12__11__7__1__9___Fit____Test_001____2023_05_29___100900_>\nusername = 'frugal@latido.com.np', password = 'Test@123'\ncontactNo = '9881012071', name = 'abced Test 1', email = 'testuser171@mail.com'\ndob = '2023-05-28', address = 'Siliguri', gender = 'male'\nproductname = 'Ek (male)', leatherProfile = 'Black Sheep', leatherSize = '38'\nhardware = 'Silver', lining = 'Black', polyfill = 'No', size = '40'\narmhole = 5.6, height = 4.4, shoulder = 5, weight = 46, length = 8, arms = 12\nhips = 11, chest = 7, waist = 1, sleeves = 9, bodytype = 'Fit'\nremark = 'Test 001', date = '2023-05-29', OTP = 100900\n\n    @allure.title(f\"Create a new Order for new Customer\")\n    @allure.story(\"Create an Order for new Customer \")\n    @allure.severity(allure.severity_level.NORMAL)\n    @data(*Utils.read_xlsx(\"../TestData/OrderTestData/orderTestCase.xlsx\", \"forNewCustomer\"))\n    @unpack\n    def test_create_new_user_order(self, username, password, contactNo, name, email, dob, address, gender, productname,\n                                   leatherProfile, leatherSize, hardware, lining, polyfill, size, armhole, height, shoulder, weight,\n                                   length, arms, hips, chest, waist, sleeves, bodytype, remark, date, OTP):\n        self.LoginMethod.nativelogin(self.driver, username, password)\n        try:\n            self.assertEqual(\n                self.Order.create_new_user_order(contactNo, name, email, dob, address, gender, productname,\n                                                 leatherProfile, leatherSize, hardware, lining, polyfill, size, armhole, height,\n                                                 shoulder, weight, length, arms, hips, chest, waist, sleeves, bodytype, remark,\n                                                 date, OTP),\n                \"Order Succesfully Created\", msg=\"Verification Failed to Create a new Order.\")\n        # Checking if assertion failed\n        except (NoSuchElementException, AssertionError, TimeoutException) as e:\n            allure.attach(self.driver.get_screenshot_as_png(), name=\"Creating new Order for new customer.\",\n                          attachment_type=AttachmentType.PNG)\n            self.log.error(f\"Assertion failed here while finding element. {str(e)}\")\n>           raise e\n\nTestOrder.py:80: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nTestOrder.py:69: in test_create_new_user_order\n    self.assertEqual(\nC:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.3.3\\plugins\\python-ce\\helpers\\pycharm\\teamcity\\diff_tools.py:33: in _patched_equals\n    old(self, first, second, msg)\nE   AssertionError: None != 'Order Succesfully Created' : Verification Failed to Create a new Order.","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_unittest_setUpClass_fixture_TestOrder","time":{"start":1685021911346,"stop":1685021911346,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"stepsCount":0,"hasContent":false}],"testStage":{"status":"failed","statusMessage":"AssertionError: None != 'Order Succesfully Created' : Verification Failed to Create a new Order.","statusTrace":"self = <TestsCases.TestOrder.TestOrder testMethod=test_create_new_user_order_1___frugal_latido_com_np____Test_123____98810120...p____38____Silver____Black____No____40___5_6__4_4__5__46__8__12__11__7__1__9___Fit____Test_001____2023_05_29___100900_>\nusername = 'frugal@latido.com.np', password = 'Test@123'\ncontactNo = '9881012071', name = 'abced Test 1', email = 'testuser171@mail.com'\ndob = '2023-05-28', address = 'Siliguri', gender = 'male'\nproductname = 'Ek (male)', leatherProfile = 'Black Sheep', leatherSize = '38'\nhardware = 'Silver', lining = 'Black', polyfill = 'No', size = '40'\narmhole = 5.6, height = 4.4, shoulder = 5, weight = 46, length = 8, arms = 12\nhips = 11, chest = 7, waist = 1, sleeves = 9, bodytype = 'Fit'\nremark = 'Test 001', date = '2023-05-29', OTP = 100900\n\n    @allure.title(f\"Create a new Order for new Customer\")\n    @allure.story(\"Create an Order for new Customer \")\n    @allure.severity(allure.severity_level.NORMAL)\n    @data(*Utils.read_xlsx(\"../TestData/OrderTestData/orderTestCase.xlsx\", \"forNewCustomer\"))\n    @unpack\n    def test_create_new_user_order(self, username, password, contactNo, name, email, dob, address, gender, productname,\n                                   leatherProfile, leatherSize, hardware, lining, polyfill, size, armhole, height, shoulder, weight,\n                                   length, arms, hips, chest, waist, sleeves, bodytype, remark, date, OTP):\n        self.LoginMethod.nativelogin(self.driver, username, password)\n        try:\n            self.assertEqual(\n                self.Order.create_new_user_order(contactNo, name, email, dob, address, gender, productname,\n                                                 leatherProfile, leatherSize, hardware, lining, polyfill, size, armhole, height,\n                                                 shoulder, weight, length, arms, hips, chest, waist, sleeves, bodytype, remark,\n                                                 date, OTP),\n                \"Order Succesfully Created\", msg=\"Verification Failed to Create a new Order.\")\n        # Checking if assertion failed\n        except (NoSuchElementException, AssertionError, TimeoutException) as e:\n            allure.attach(self.driver.get_screenshot_as_png(), name=\"Creating new Order for new customer.\",\n                          attachment_type=AttachmentType.PNG)\n            self.log.error(f\"Assertion failed here while finding element. {str(e)}\")\n>           raise e\n\nTestOrder.py:80: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\nTestOrder.py:69: in test_create_new_user_order\n    self.assertEqual(\nC:\\Program Files\\JetBrains\\PyCharm Community Edition 2022.3.3\\plugins\\python-ce\\helpers\\pycharm\\teamcity\\diff_tools.py:33: in _patched_equals\n    old(self, first, second, msg)\nE   AssertionError: None != 'Order Succesfully Created' : Verification Failed to Create a new Order.","steps":[],"attachments":[{"uid":"a8a3a7b38f484914","name":"Creating new Order for new customer.","source":"a8a3a7b38f484914.png","type":"image/png","size":61057},{"uid":"d7acde489b2c498d","name":"log","source":"d7acde489b2c498d.txt","type":"text/plain","size":11539}],"parameters":[],"attachmentsCount":2,"shouldDisplayMessage":true,"stepsCount":0,"hasContent":true},"afterStages":[{"name":"_unittest_setUpClass_fixture_TestOrder::0","time":{"start":1685024388837,"stop":1685024388837,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"attachmentsCount":0,"shouldDisplayMessage":false,"stepsCount":0,"hasContent":false}],"labels":[{"name":"story","value":"Create an Order for new Customer "},{"name":"feature","value":"Order List"},{"name":"severity","value":"normal"},{"name":"suite","value":"TestOrder"},{"name":"subSuite","value":"TestOrder"},{"name":"host","value":"0day"},{"name":"thread","value":"4200-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"TestOrder"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Product defects","matchedStatuses":[],"flaky":false}],"tags":[]},"source":"d141263e4b68c05e.json","parameterValues":[]}